docs/user_guide/model_repository.md

```diff
@@ -118,7 +118,25 @@ $ tritonserver --model-repository=gs://bucket/path/to/model/repository ...
 
 When using Google Cloud Storage, the
 [GOOGLE_APPLICATION_CREDENTIALS](https://cloud.google.com/docs/authentication/application-default-credentials#GAC)
-environment variable should be set.
+environment variable should be set and contains the location of a credential
+JSON file. If no credential is provided, Triton will use credentials from the
+[attached service account](https://cloud.google.com/docs/authentication/application-default-credentials#attached-sa)
+providing a value for the
+[Authorization HTTP header](https://googleapis.dev/cpp/google-cloud-storage/1.42.0/classgoogle_1_1cloud_1_1storage_1_1oauth2_1_1ComputeEngineCredentials.html#a8c3a5d405366523e2f4df06554f0a676) 
+can be obtained. If not obtainable, anonymous credential will be used.
+
+To access buckets with anonymous credential (also known as public bucket), the
+bucket (and objects) should have granted get and list permissions to all users.
```

Comment:
use code block for `get` and `list` words.

Reply:
Yes, updated.